lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -326,7 +365,7 @@ def _resume_tunnel_flows(self, precedence:int, i_teid:int,
         # discard downlink Tunnel for LOCAL port
         if not ue_ip_adr:
```

Comment:
same, lets do required arg checks in the beginning because otherwise its kind of doing partial work and that will just be a pain to debug, and then the code will be clean as well because we can safely assume all args are valid

Reply:
Yes. Added check beginning of the received grpc message.